src/dec/entis/eri_image_decoder.cc

```diff
@@ -41,8 +42,8 @@ static image::EriHeader read_header(
 
     const s32 width         = input_stream.read_le<u32>();
     const s32 height        = input_stream.read_le<u32>();
-    header.width            = std::abs(width);
-    header.height           = std::abs(height);
+    header.width            = std::fabs(width);
+    header.height           = std::fabs(height);
```

Comment:
`#include <cstdlib>` fixes this without having to change `std::abs` to `std::fabs`.